packages/arb-token-bridge-ui/src/util/CommonUtils.ts

```diff
@@ -1,3 +1,33 @@
+let localStoragePromise = Promise.resolve()
+
+export function addToLocalStorageObjectSequentially({
```

Comment:
We use it when we call multiple local storage setters in parallel, then it will work like a queue system instead of overwriting each others values